java/src/main/java/com/powsybl/dataframe/network/NetworkDataframeMapperBuilder.java

```diff
@@ -28,10 +29,21 @@ public NetworkDataframeMapperBuilder() {
         this.addProperties = false;
     }
 
+    @FunctionalInterface
+    public interface TriFunction<T, U, I> {
```

Comment:
We should make that class more specific and explicit:
 - give it an explicit name like `ItemGetter`
 - remove the `T` and `U` parameters which will always be `Network` and `UpdatingDataframe`
 - also name the method more explicitly like `getItem`
 - name the parameters more explicitly, for example `lineNumber` for the last parameter